### PR TITLE
[sdkv2] add indexer query getAccountOwnedTokensByTokenId

### PR DESCRIPTION
**Pull request by 0xjinn**
### Description

Add getOwnedTokensByTokenData over to sdk v2

Note: 
1. Test is not included yet, will wait until we have Token client/class to wrap up the test.
2. Another account token query `getTokenOwnedFromCollectionNameAndCreatorAddress` is not included, because it relies on getCollectionAddress query, which Oliver is working on. 

### Test Plan

pnpm fmt
pnpm test
pnpm lint

### CODE REVIEW

**Comment by 0xjinn** [comment]
Current dependencies on/for this PR:
* main
  * **PR #29** <a href="https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/29" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/aptos-labs/aptos-ts-sdk/29?utm_source=stack-comment).

**Review by 0xmaayan — COMMENTED** [review]
we also need to add the api/internal functions...?

File: src/types/generated/operations.ts [hunk 1]
```diff
@@ -1,336 +1,96 @@
-import * as Types from "./types";
+import * as Types from './types';
```
Comment (0xmaayan) [inline src/types/generated/operations.ts:1]: did you run fmt or something on this file? not sure why it changed

Comment (0xjinn) [reply src/types/generated/operations.ts:1]: Yep, I generally run `pnpm fmt` on the whole repo.

**Comment by 0xjinn** [comment]
> we also need to add the api/internal functions...?

oops, good catch.

File: src/types/generated/operations.ts (hunk 1 above)
Comment (0xmaayan) [reply src/types/generated/operations.ts:1]: oh I think it ignores `generated` files, not sure why it changed here

Comment (0xjinn) [reply src/types/generated/operations.ts:1]: Hmm maybe `pnpm run indexer-codegen` auto reformats?

Comment (0xmaayan) [reply src/types/generated/operations.ts:1]: ohh maybe....

Comment (gregnazario) [reply src/types/generated/operations.ts:1]: Yeah, pnpm run indexer-codegen reformats, maybe we just apply autoformat to even generated files (just ignoring lints of course)

**Comment by 0xjinn** [comment]
Didn't realize this query already been ported over with a different name? I will close this PR - https://github.com/aptos-labs/aptos-ts-sdk/blob/main/src/internal/queries/getAccountOwnedTokensByTokenData.graphql#L2-L11

**Pull request closed without merging**